rllib/evaluation/rollout_worker.py

```diff
@@ -177,6 +177,7 @@ def __init__(
             fake_sampler: bool = False,
             spaces: Optional[Dict[PolicyID, Tuple[gym.spaces.Space,
                                                   gym.spaces.Space]]] = None,
+            _use_trajectory_view_api: bool = True,
```

Comment:
Pass this in explicitly now into RolloutWorker (was derived from `policy_config` before, which is problematic as this could be a partial config dict)